app/assets/stylesheets/modules/_landing.scss

```diff
@@ -0,0 +1,109 @@
+.hero {
+  background-color: $white;
+
+  &-title {
+    color: $dark-gray;
+    margin-bottom: em(5);
+    text-align: center;
+  }
+
+  &-subtitle {
+    font-style: italic;
+    font-weight: 300;
+    margin-top: em(5);
+    text-align: center;
+  }
+
+  &-main-text {
+    padding: 0 0 em(50);
+    text-align: center;
+    @include span-columns(8 of 12);
+    @include shift(2);
+  }
+
+}
+
+.organization {
+  background-color: $base-background-color;
+
+  &-title {
+    color: $white;
+    text-align: center;
+  }
+
+  &-subtitle {
+    color: $white;
+    font-weight: 400;
+    padding-bottom: 1em;
+    text-align: center;
+
+    & a {
+      border-bottom: 1px solid $white;
+      color: $white;
+      display: inline-block;
+      line-height: 0.97em;
+      margin: 0.2em auto;
+      text-shadow:/* stroke around*/
+        0 0px 1px $teal,
+        0 0px 1px $teal,
+        0 0px 2px $teal,
+        0 0px 2px $teal,
+        0 0px 2px $teal,
+        0 0px 2px $teal,
+        0 0px 2px $teal,
+        0 0px 2px $teal,
+        0 0px 2px $teal,
+        0 -5px 0 $teal,/* clear vertically*/
+        0 -1px 0 $teal,
+        0 -2px 0 $teal,
+        0 -2px 0 $teal,
+        0 -4px 0 $teal,
+        0 -5px 0 $teal,
+        0 -6px 0 $teal
+        ;
+
+      &:hover {
+        border-bottom: 1px solid $action-color;
+        color: $action-color;
+      }
+    }
+  }
+
+  &-card {
+    background-color: $white;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets